meson.build

```diff
@@ -46,7 +46,7 @@ cdata = configuration_data()
 cdata_asm = configuration_data()
 
 # Include directories
-dav1d_inc_dirs = include_directories(['.', 'include/dav1d', 'include'])
+dav1d_inc_dirs = include_directories(['.', 'src/include/dav1d', 'src/include'])
```

Comment:
We do need to make sure that `git cherry-pick` works on `dav1d` commits that touch C files.  I'm not sure if git can handle this rename (it might, not sure, though).  If it can't, we should leave the C files where they are and just move the Rust files.

Reply:
```
djc-2021 reroot rav1d $ git cherry-pick 7d4b789f55389dad1820d6caf6a650038dad06e2
Auto-merging src/include/common/attributes.h
[reroot 36236a45] Mark C globals with small code model
 Author: Pranav Kant <prka@google.com>
 Date: Fri Feb 21 15:55:00 2025 +0000
 2 files changed, 13 insertions(+)
```

7d4b789 is the latest commit from dav1d/master that touches `include/`. Seems okay?